x/auth/vesting/types/msgs.go

```diff
@@ -163,7 +163,7 @@ func (msg MsgCreateClawbackVestingAccount) ValidateBasic() error {
 // The dest address may be nil - defaulting to the funder.
 //nolint:interfacer
 func NewMsgClawback(funder, addr, dest sdk.AccAddress) *MsgClawback {
-	var destString string
+	destString := ""
```

Comment:
nit: I'd prefer to keep the `var` instantiation 👍 

this goes for any zero value variable